cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/MetricFetcherManager.java

```diff
@@ -79,7 +80,8 @@ public MetricFetcherManager(KafkaCruiseControlConfig config,
    * @param metricDef the metric definitions.
    * @param time        The time object.
    * @param dropwizardMetricRegistry The Metric Registry object.
-   * @param fetchers    A list of metric fetchers.
+   * @param brokerCapacityConfigResolver The resolver for retrieving broker capacities.
+   * @param fetcher Metric fetcher or {@code null} to create one using {@link KafkaCruiseControlConfig#METRIC_SAMPLER_CLASS_CONFIG}.
```

Comment:
Could we use `sampler` for this parameter? `fetcher` reminds me of `MetricFetcher`.